mex-wholebodymodel/matlab/utilities/whatname.m

```diff
@@ -0,0 +1,163 @@
+function name = whatname(name_group,k)
+%% whatname
+%  It assigns titles to graphics according to robot's joints distribution.
+%  The inputs are the robot link name (torso, r_arm, l_arm,...) and the interested 
```

Comment:
There is already a convention of the iCub part names, that are the one used in the controlboards, see http://wiki.icub.org/wiki/ICub_joints . If possible, use `right_arm` and `left_arm` in place of `l_arm` and `r_arm`.